flang/test/Driver/masm.f90

```diff
@@ -0,0 +1,13 @@
+// RUN: %flang -target x86_64-unknown-linux -masm=intel -S %s -### 2>&1 | FileCheck --check-prefix=CHECK-INTEL %s
```

Comment:
1. Use Fortran style comments :)
2. Missing requires?

Reply:
`--target=` for new tests

Reply:
1. Done, I forgot this wasn't actually parsing the file. So I've removed the function as well, it's not needed.
2. Don't need requires if all we're doing is using the triple name, I checked that by building without x86 and the test still works.
3. Used --target= for the run lines.